dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/newsession/NewSessionPopup.java

```diff
@@ -17,10 +17,19 @@
 
 import com.spectralogic.dsbrowser.gui.util.Popup;
 
+import javax.inject.Inject;
+import javax.inject.Singleton;
 import java.util.ResourceBundle;
 
-public final class NewSessionPopup {
-    public static void show(final ResourceBundle resourceBundle) {
+@Singleton
+public class NewSessionPopup {
```

Comment:
Is there a reason you removed the `final` modifier from this class?

Reply:
The class is now instantiated which reduces the need to pass around the resource bundle.